.github/actions/do_build_ock/action.yml

```diff
@@ -98,69 +95,80 @@ inputs:
     description: "Disable unitcl vecz checks"
     default: OFF
   gtest_launcher:
-    description: "Googletest suite launcher command"
+    description: "Googletest suite launcher command (default ubuntu)"
     default: "/usr/bin/python;-u;${{ github.workspace }}/scripts/gtest-terse-runner.py"
+  build_32_bit:
+    description: "32-bit building"
+    default: OFF
+  shell_to_use:
+    description: "which shell to use (default bash)"
+    default: bash
   c_compiler:
-    description: "C compiler"
+    description: "C compiler (default gcc)"
     default: "gcc"
   cxx_compiler:
-    description: "C++ compiler"
+    description: "C++ compiler (default g++)"
     default: "g++"
 
 runs:
-  # We don't want a new docker just a list of steps, so mark as composite
+  # we don't want a new docker just a list of steps, so mark as composite
```

Comment:
not sure why you've lowercased this, comments above have capitals.

Reply:
Will restore in the next PR.